check-and-talk/src/components/ProfileEdit.js

```diff
@@ -0,0 +1,31 @@
+import React, {Component} from 'react';
+import './ProfileEdit.css'
+import Profile_red from '../assets/Profile_Red.svg';
+
+class ProfileEdit extends Component {
+    render() {
+        return (
+            <div className='ProfileEdit-style'>
+                
+                <div className='Profile-img'>
+                    <img src={Profile_red} width={269}></img>
+                </div>
+                <div className='ProfileEdit-color'>
+                    <div className='color-info'>
+                        <div className='red'></div>
+                        <div className='yellow'></div>
+                        <div className='green'></div>
+                        <div className='blue'></div>
+                        <div className='purple'></div>
```

Comment:
render 메서드 안에서
`const colorList = ["red", "yellow", "green", "blue", "purple"];`
선언해주고
```suggestion
                        {colorList.map((v) => (<div className={v}></div>))}
````
이런식으로 코드를 짜면 관리하기 편할 것 같습니다